src/test/groovy/GithubWorkflowRunTests.groovy

```diff
@@ -259,4 +259,36 @@ View this run on GitHub: https://github.com/owner/repo/actions/runs/1441316856
     def expected = [id: 1441316856, status: "completed"]
     assertTrue(expected.id == result.id && expected.status == result.status)
   }
+
+  @Test
+  void test_getWorkflowRun_works_properly_with_ORG_NAME_and_REPO_NAME_set() throws Exception {
+    env.REPO_NAME = 'foo'
+    env.ORG_NAME = 'org'
+    def result = script.getWorkflowRun(runId: 1441316856, repo: "owner/repo")
+    def expected = [id: 1441316856, status: "completed"]
+    assertTrue(expected.id == result.id && expected.status == result.status)
+    env.REPO_NAME = 'repo'
+    env.ORG_NAME = 'owner'
```

Comment:
Is this needed? IIRC, if `env.NAME` is defined is already set, so  L265 and L266 already set the values.

Reply:
yes, it's important because it checks different case - when repo argument is not specified and  repo info is taken from these environment variables

Reply:
I don't see any assertion for those particular environment variables, therefore I don't see why we need them twice.
If they are needed, can you add the required assertion?

Reply:
Values from these variables used to create gh api command line. Then this command like checked in  *ghClosure* closure where encoded only 3 cases for testing *gh api* calls. All these cases are for repo value *owner/repo*. So to use one of  them for checking this case I need to set these variables to 
```
    env.REPO_NAME = 'repo'
    env.ORG_NAME = 'owner'
```
This is all cases which currently are in ghClosure:
```
      if (args.command == "api repos/owner/repo/actions/runs/1441316856") {
        return '{"id": 1441316856, "status": "completed"}'
      }
      if (args.command == "api repos/owner/repo/actions/runs/1441334029") {
        return '{"message": "Not Found"}'
      }
      if (args.command == "api repos/owner/repo/actions/runs/1441316123") {
        return '{"id": 1441316123, "status": "queued"}'
      }
```

Reply:
Gotcha, thanks for the clarification